server/routes/index.ts

```diff
@@ -21,41 +23,24 @@ export default function routes(service: Services): Router {
     res.render('pages/index')
   })
 
-  get('/events', async (req, res, next) => {
-    const { user } = res.locals
-    const systemToken = await hmppsAuthClient.getSystemClientToken(user.username)
-    const incidentReportingApi = new IncidentReportingApi(systemToken)
-
-    const response = await incidentReportingApi.getEvents({
-      prisonId: user.activeCaseLoadId,
-      // eventDateFrom: new Date('2024-07-30'),
-      // eventDateUntil: new Date('2024-07-30'),
-      // sort: ['eventDateAndTime,ASC'],
-    })
-
-    // No authorisation at this point, no data shown in production
-    if (config.environment === 'prod') {
-      response.content = []
-    }
-
-    res.render('pages/events', { response })
-  })
-
   get('/incident/:id', async (req, res, next) => {
     const { id } = req.params
 
     const { user } = res.locals
     const systemToken = await hmppsAuthClient.getSystemClientToken(user.username)
     const incidentReportingApi = new IncidentReportingApi(systemToken)
+    const manageUsersApiClient = new ManageUsersApiClient()
 
     // No authorisation at this point, no data shown in production
     if (config.environment === 'prod') {
       throw new NotFound()
     }
 
     const event = await incidentReportingApi.getEventById(id)
+    const inputUsernames = event.reports.map(report => report.modifiedBy)
+    const nameMappings = await makeUsernameLookup(manageUsersApiClient, systemToken, inputUsernames)
```

Comment:
We could call this `staffLookup` for consistency with the variable for the `prisonersLookup` :)

Reply:
this is not needed anymore due to the new `userService.getUsers` function